### PR TITLE
Add more ScalarFuncSig types for TiDB builtin-UDFs push down

### PR DESCRIPTION
**Pull request by bb7133**
This is a prerequisites of TikV #2918

### CODE REVIEW

**Comment by bb7133** [comment]
Simply added following types in expression.proto:

IntDivideInt
IntDivideDecimal
ModReal
ModDecimal
ModInt
BitCount
LeftShift
RightShift
Greatest(*Types)/Least(*Types)
IntervalInt
IntervalReal
RegexpBinarySig
RegexpSig
BitLength

@zz-jason @coocood PTAL, thanks

File: proto/expression.proto [hunk 1]
```diff
@@ -196,6 +196,11 @@ enum ScalarFuncSig {
 	MultiplyInt = 210;
 	DivideReal = 211;
 	DivideDecimal = 212;
+    IntDivideInt = 213;
```
Comment (zz-jason) [inline proto/expression.proto]: please keep the code indent

**Comment by zz-jason** [comment]
to https://github.com/pingcap/tikv/issues/2918

**Comment by bb7133** [comment]
@zz-jason  Fixed the indent problem, thanks

**Comment by zz-jason** [comment]
@XuHuaiyu @winoros @coocood PTAL

**Pull request merged**